.env

```diff
@@ -12,6 +12,8 @@ enable_email_server=false
 save_images_to_db=true
 compreface_api_java_options=-Xmx4g
 compreface_admin_java_options=-Xmx1g
+max_file_size=10MB
+client_max_body_size=10M
```

Comment:
what is the difference between these two variables?

Reply:
max_file_size - sets the total request size for a multipart/form-data.
client_max_body_size - sets the maximum allowed size of the client request body.

Should I join them in one?

Reply:
no, let's leave them separate.
I just don't understand, why then in [java/admin/src/main/resources/application.yml](https://github.com/exadel-inc/CompreFace/pull/778/files#diff-d0bf48421385c221ee3c77956f0ea941b80f5cefb5d91678cb8e20435274ff52) you assign the same variable:
```     
max-file-size: ${MAX_FILE_SIZE:10MB}
max-request-size: ${MAX_FILE_SIZE:10MB}```

Reply:
I thought it can be ok in our case. I will split it